services/gateway/README.md

```diff
@@ -1,19 +1,25 @@
 # Gateway
 
-The gateway service is an ...
+The gateway service is responsible for passing requests to the storage providers. Other services never talk to the storage providers directly but will always send their requests to the `gateway` service.
```

Comment:
🧡

Reply:
I'd say "via" is more fitting, although it's implied that the requests will reach the storage providers in the end.